src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.css"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__item-img">
+        <a href="#APPLE_A1419_iMac_27">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE_A1419_iMac_27"
+          />
+        </a>
+      </div>
+
+      <div class="card__item-content">
+        <h3 class="card__item-name">
```

Comment:
take a look, there is a default h3 margin applied by the browser:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/f7e9f67e-2cc4-42d3-adc0-20b51d9f7ff8)

It makes the space between image and title too big. Reset margin for h3 in page.scss as you do for `p`, or/and define the correct margin for it using class

Reply:
Do I should add a button?